packages/v4/src/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -174,11 +174,11 @@ Use a compound expandable table when:
 * It would not make sense to combine all of this information into a single, simple expansion.
 
 ### Actionable table
-The actionable table provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/design-guidelines/usage-and-behavior/toolbar).
+The actionable table provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/components/toolbar/design-guidelines).
 
 <img src="./img/actionable-table.png"  alt="Actionable table"  width="1174"/> 
 
-1. **[Bulk selection](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/design-guidelines/usage-and-behavior/bulk-selection) for more information. 
+1. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information. 
```

Comment:
no bulk select guidelines

Reply:
These are going to be added back soon. I'm fine with the link being broken for now.